### ISSUE TITLE
Feature request: support copyfile from local to container

### ISSUE THREAD

**Issue by wanghaoran1988** [GoogleCloudPlatform/container-structure-test#73]
When testing some image, we need copy some local file to the image and verify whether it work well or not.

The "Setup" field was a set of command running in the container before we testing the image, it will better if we support "COPY" direc in the "Setup" field.

**Comment by goutamtadi1** [comment]
I would like to see this feature get in soon. We would like to run some tests this way, so that we can avoid copying test scripts into the image. This helps us keeping the images concise and doing only one thing. Please prioritize. Thanks.

**Comment by donmccasland** [comment]
Agreed.  We'll need to dust off the work in #82 to get this addressed.  Iceboxing and adding 'help wanted'

**Comment by coopernetes** [comment]
Should be possible now via #327 and v1.15
